app/src/main/java/net/squanchy/typeface/TypedArrayHelper.kt

```diff
@@ -0,0 +1,31 @@
+package net.squanchy.typeface
```

Comment:
There is a `net.squanchy.support` package, this should be in there! `net.squanchy.support.font` seems an appropriate package name.